apps/src/code-studio/pd/professional_learning_landing/csPrinciplesAndDiscoveriesSection.jsx

```diff
@@ -1,16 +1,64 @@
 /** Section for CS Principles and CS Discoveries */
 
 import React from 'react';
+import {TwoPartBanner, imageStyle, textStyle, leftStyle, rightStyle} from './twoPartBanner';
+import {Button} from 'react-bootstrap';
 
 const CsPrinciplesAndDiscoveriesSection = React.createClass({
   propTypes: {
+    coursesCompleted: React.PropTypes.arrayOf(React.PropTypes.string),
+    lastWorkshopSurveyUrl: React.PropTypes.string
+  },
 
+  renderCompletedCourseBox() {
+    if (this.props.lastWorkshopSurveyUrl) {
+      return (
+        <div id="cspStartSurvey">
+          <h3>
+            Share your feedback
+          </h3>
+          <p>
+            Thank you for completing a Code.org professional learning owrkshop! Please complete this survey so that we
+            may improve your experience.
+          </p>
```

Comment:
hmm - now that I think about it, I need it immediately. People will come back to this page after they've completed the survey and I don't want there to be confusion.

The problem is that I don't want to do a big query into the pegasus db every time someone hits this form looking for fresh survey completions. 

Is there a controller for these form responses? What if we updated the Dashboard DB on submission?